broker-plugin/plugin/src/main/sh/init-broker.sh

```diff
@@ -16,6 +16,8 @@ cp -r ${BROKER_PLUGIN_DIR}/jmx_exporter/jmx_prometheus_javaagent-0.1.0.jar $BROK
 cp -r ${BROKER_PLUGIN_DIR}/lib $BROKER_CUSTOM
 cp -r ${BROKER_PLUGIN_DIR}/bin $BROKER_CUSTOM
 
+source /opt/run-java/dynamic_resources.sh > /dev/null
```

Comment:
Does the init-container have the same resources available as the actual container?  If so, it would be fine to move this into the init container.  The downstream broker/launch.sh will also run this.

Reply:
@vbusch you are right.   My approach was flaw as I was generating the JVM arguments from within the wrong container.  If the base image diverged the results would have been wrong and as you pointed out, settings based on resources could have been wrong.

 I took an alternative approach instead.